modules/http/src/main/java/org/mule/module/http/internal/request/grizzly/IOStrategyTransportCustomizer.java

```diff
@@ -40,7 +40,7 @@ public IOStrategyTransportCustomizer(String threadNamePrefix, Integer maxWorkerP
     @Override
     public void customize(TCPNIOTransport transport, FilterChainBuilder filterChainBuilder)
     {
-        transport.setIOStrategy(FlowWorkManagerIOStrategy.getInstance());
+        transport.setIOStrategy(WorkerThreadIOStrategy.getInstance());
```

Comment:
- Where is the size of this pool configured, whats default?
- We should still hand off to Flow pool at some point.